src/hotspot/share/oops/symbol.cpp

```diff
@@ -143,6 +144,15 @@ int Symbol::index_of_at(int i, const char* substr, int substr_len) const {
   return -1;
 }
 
+bool Symbol::is_star_match(const char* star_pattern) const {
```

Comment:
So this variable name should be "pattern" not "star_pattern" since it may not have a star.

Reply:
Fixed.